.pre-commit-config.yaml

```diff
@@ -0,0 +1,62 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0  # Use the ref you want to point at
+    hooks:
+    -   id: trailing-whitespace
+    -   id: no-commit-to-branch
+        args: ['--branch', 'master']
+    -   id: check-merge-conflict
+    -   id: double-quote-string-fixer
+    -   id: requirements-txt-fixer
+    -   id: check-yaml
+    -   id: check-docstring-first
+    -   id: end-of-file-fixer
+    -   id: name-tests-test
+    -   id: check-ast
+    -   id: debug-statements
+    # -   id: ...
+
+-   repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
+    hooks:
+    -   id: flake8
+
+-   repo: https://github.com/asottile/add-trailing-comma
+    rev: v2.4.0
+    hooks:
+    -   id: add-trailing-comma
+
+-   repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.3.1
+    hooks:
+    - id: forbid-crlf
+    - id: remove-crlf
+    - id: forbid-tabs
+    - id: remove-tabs
+      args: [--whitespaces-count, '2']  # defaults to: 4
+    - id: insert-license
+      files: \.groovy$
+      args:
+        - --license-filepath
```

Comment:
I see that this YAML file uses inconsistent indents for lists. Here's a config that I usually use to enforce consistent style: https://github.com/sphinx-contrib/sphinxcontrib-towncrier/blob/master/.yamllint.
You can add it as a separate commit and integrate it along with the corresponding hook like in this example https://github.com/sphinx-contrib/sphinxcontrib-towncrier/blob/master/.pre-commit-config.yaml. Feel free to also borrow other hooks listed there.

Reply:
Resolved.